Gemfile

```diff
@@ -126,9 +123,10 @@ gem 'connection_pool'
 gem 'dalli'
 gem 'kgio', '2.10.0'
 
+gem "mimemagic", "0.3.10"
```

Comment:
Previously we kept mimemagic, because it was required by paperclip even though it was no longer required by Active Storage (#3988). mimemagic is no longer a dependency of either, so let's eventually get rid of it. Can you add a TODO here?

Reply:
I'm not entirely sure we can get rid of it because of how we're using it for downloads:  https://github.com/otwcode/otwarchive/pull/3929#discussion_r800303934

Reply:
I think we can, Marcel has similar lookup methods we can use as replacements. We don't need to do that now though, hence a TODO.